### PR TITLE
Create agentAutoUpdate.sh

### PR DESCRIPTION
**Pull request by tanmaygarg-oracle**
Script required to auto update

### CODE REVIEW

File: OracleIdentityGovernance/samples/scripts/agentAutoUpdate.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# Author: OIG Development
+#
+# Description: Script for upgrading agent
+#
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+cd "$1" || exit
+mkdir -p newpackage || true
+#source $1/data/conf/config.properties
```
Comment (rishiagarwal-oracle) [inline OracleIdentityGovernance/samples/scripts/agentAutoUpdate.sh:16]: remove this

File: OracleIdentityGovernance/samples/scripts/agentAutoUpdate.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+cd newpackage || exit
+#Download upgrade cli
+wget https://objectstorage.us-ashburn-1.oraclecloud.com/p/fFvMAmluNZpv4P5dCzH7VsyJUYra5AMxhLiBSOa3AZuul4KtycxDuJtyUyWaweU4/n/idjypktnxhrf/b/agcs_ido_agent_updater/o/idm-agcs-agent-cli-upgrade.jar
+
+#Get Agent Package
+agentVersion=$(unzip -q -c  "$1"/agent-lcm/idm-agcs-agent-lcm.jar META-INF/MANIFEST.MF | grep "Agent-Version: " | awk '{print $2}' | tr -d '\n' | tr -d '\r')
```
Comment (rishiagarwal-oracle) [inline OracleIdentityGovernance/samples/scripts/agentAutoUpdate.sh:23]: fix the directory

**Review by rishiagarwal-oracle — CHANGES_REQUESTED** [review]
please incorporate the comments

File: OracleIdentityGovernance/samples/scripts/agentAutoUpdate.sh (hunk 2 above)
Comment (tanmaygarg-oracle) [reply OracleIdentityGovernance/samples/scripts/agentAutoUpdate.sh]: fixed

**Comment by rishiagarwal-oracle** [comment]
LGTM !

**Review by rishiagarwal-oracle — APPROVED** [review]

**Comment by tanmaygarg-oracle** [comment]
@Djelibeybi Please review

**Comment by Djelibeybi** [comment]
You need to assign PRs to me when you want me to review the.

**Pull request merged**